tools/structure/src/main.py

```
@@ -160,35 +168,74 @@ def run(
             )
-            llm_profile_id = self.get_exec_metadata.get(SettingsKeys.LLM_PROFILE_ID)
-            llm_profile_to_override = None
+            tool_metadata = exported_tool[SettingsKeys.TOOL_METADATA]
+
+            # Check if this is actually an agentic project
+            if self._is_agentic_project(tool_metadata):
+                is_agentic = True
+                self.stream_log(
+                    f"Detected agentic project: {tool_metadata.get('name', prompt_registry_id)}"
+                )
+        except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)